.github/workflows/pipeline.yml

```diff
@@ -42,3 +43,31 @@ jobs:
         fail_ci_if_error: true
         files: ./coverage.txt
         verbose: true
+  deploy:
+    if: startsWith(github.ref, 'refs/tags/v')
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Prepare
+        id: prep
+        run: |
+          DOCKER_IMAGE=quay.io/kohlstechnology/blackbox-helloworld-responder
+          VERSION=${GITHUB_REF#refs/tags/}
+          TAGS="${DOCKER_IMAGE}:${VERSION}"
+          echo ::set-output name=tags::${TAGS}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to quay.io
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.QUAY_USER }}
+          password: ${{ secrets.QUAY_TOKEN }}
+          registry: quay.io
+      - name: Build and push to quay.io
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.prep.outputs.tags }}
```

Comment:
Probably won't update the `latest` tag in quay.io, but I'm ok with that for now. I just want to see if it works or not. A future PR can update the pipeline to also push the `latest` tag.

Reply:
I don't think so either, but fair.